graf2d/gpad/src/TPad.cxx

```diff
@@ -5234,6 +5234,12 @@ void TPad::RangeAxis(Double_t xmin, Double_t ymin, Double_t xmax, Double_t ymax)
    RangeAxisChanged();
 }
 
+void TPad::RangeAxis()
```

Comment:
I must have missed something but I don't understand why you need this intermediate method which only call RangeAxisChanged(); ?